### ISSUE TITLE
can you add this feature

### ISSUE THREAD

**Issue by metapodcod** [alexal1/Insomniac#99]
can you make a feature like here https://github.com/InstaPy/instapy-quickstart/blob/master/quickstart_templates/good_commenting_strategy_and_new_qs_system.py

**Comment by Fadude** [comment]
Added on v3.6.0. Closing.

**Comment by metapodcod** [comment]
@alexal1  @Fadude  where did you add this feature, it does not appear in the wiki 
Can you show me an example usage ?

**Comment by Fadude** [comment]
Version 3.6.0, edit and use this config file:
https://github.com/alexal1/Insomniac/blob/master/config-examples/interact/interact-likes-follow-story-comment.json

Take a look at the release notes of 3.6.0 :)

**Comment by metapodcod** [comment]
it looks scary to me

I do not have code knowledge, I could not understand where to write hashtags, where to put comments database, I will write, I do not understand anything but still thanks this adding feature

**Comment by metapodcod** [comment]
this is not what i want 
goes to these shared posts and follows people who like them

https://github.com/InstaPy/instapy-quickstart/blob/master/quickstart_templates/good_commenting_strategy_and_new_qs_system.py  Here, whoever shared the content follows and comments on her post.

@Fadude

**Comment by Fadude** [comment]
@metapodcod So its interact-by-post-likers?

**Comment by yuceltoluyag** [comment]
https://github.com/InstaPy/instapy-quickstart/blob/master/quickstart_templates/good_commenting_strategy_and_new_qs_system.py "instapy" works like this: navigate to given tags, receive posts, like posts, comment on that post and move to next content,

for example  https://www.instagram.com/explore/tags/turkey/  counts all content here, performs the operations I mentioned above respectively or in a mixed way  

In short, we are interested in the posts on the label, not with who liked them.

I'm sorry for my bad english

**Comment by metapodcod** [comment]
https://github.com/InstaPy/instapy-quickstart/blob/master/quickstart_templates/good_commenting_strategy_and_new_qs_system.py "instapy" works like this: navigate to given tags, receive posts, like posts, comment on that post and move to next content,

for example https://www.instagram.com/explore/tags/turkey/ counts all content here, performs the operations I mentioned above respectively or in a mixed way

In short, we are interested in the posts on the label, not with who liked them.

I'm sorry for my bad english  @Fadude

**Comment by mattr9** [comment]
> https://github.com/InstaPy/instapy-quickstart/blob/master/quickstart_templates/good_commenting_strategy_and_new_qs_system.py "instapy" works like this: navigate to given tags, receive posts, like posts, comment on that post and move to next content,
> 
> for example https://www.instagram.com/explore/tags/turkey/ counts all content here, performs the operations I mentioned above respectively or in a mixed way
> 
> In short, we are interested in the posts on the label, not with who liked them.
> 
> I'm sorry for my bad english @Fadude

YES!!!! this is such a more human way to navigate instagram. please make this.

**Comment by mattr9** [comment]
maybe to clarify the requested interaction if this helps...the idea is that you go to a hashtag and view its recent images, select X number of those images, like Y percent of them and comment on Z percent of those. rather than going to users profiles based on some list, the bot would just be liking recently posted things.

**Comment by metapodcod** [comment]
humm it's a pity that this feature has not been added yet :(  Commenting, liking and following the content shared on the hashtag

**Comment by metapodcod** [comment]
I've been waiting for a development on this issue for 1 year, is this project dead? @Fadude  @alexal1
